packages/cloudflare/README.md

```diff
@@ -92,6 +92,7 @@ There are three options available:
   Example: For `src/pages/index.astro` (static), `src/pages/company.astro` (static), `src/pages/users/faq.astro` (static) and `/src/pages/users/[id].astro` (SSR) this will produce the following `_routes.json`:
 
   ```json
+  // dist/_routes.json
```

Comment:
We should remove it because comments aren't valid in JSON, and update the file name at line 92, which seems to be wrong.

Reply:
Actually you are right. I was changing unrelated things in the docs. I'll tackle those `_routes.json` improvements in a separate PR & will focus for the docs only to the PR scope.